XDS_PROTOCOL.md

```diff
@@ -191,11 +191,15 @@ drop can't be tolerated, traffic drop could have been avoided by providing a
 CDS/EDS update with both __X__ and __Y__, then the RDS update repointing from
 __X__ to __Y__ and then a CDS/EDS update dropping __X__.
 
-In general, to avoid traffic drop:
-* Sequencing should be make before break.
-* LDS and CDS updates should arrive before the respective RDS and EDS updates.
-* CDS/EDS resources corresponding to routes in LDS/RDS should be available at
-  update.
+In general, to avoid traffic drop, sequencing of updates should follow a
+`make before break` model, wherein
+* CDS updates (if any) must always be pushed first.
+* EDS updates (if any) must arrive after CDS updates for the respective clusters.
+* LDS updates must arrive after corresponding CDS/EDS updates.
+* RDS updates related to the newly added listeners must arrive in the end.
+
+xDS updates can be pushed independently if no new clusters/routes/listeners
```

Comment:
This is too strong and not accurate I think. You can add a route that refers to existing clusters, for example, with a completely independent RDS update. Also, if you modify a cluster, it will need to go out and fetch its corresponding EDS update, see https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/envoy-users/jn3VJJBGoPo/Z_EpNSk2BgAJ. We probably need CDS/EDS warming to be implemented to make updates safe from dropping traffic (otherwise, you probably need to do a full RDS/CDS/EDS update sequence with new cluster name to avoid drops).

Reply:
should i remove it altogether?